src/pages/understand-stacks/microblocks.md

```diff
@@ -0,0 +1,97 @@
+latency. Because each Stacks block is anchored to a Bitcoin block through the [Proof-of-Transfer consensus mechanism][],
+Stacks is necessarily limited to the same block times as the Bitcoin network. Microblocks allow the Stacks blockchain to
+perform state transitions with a high degree of confidence between final settlements against Bitcoin.
+
+Microblocks are a powerful mechanism for developers to create performant, high quality applications on Stacks, while
+still inheriting the security of Bitcoin.
+
+## Transaction states
+
+The Stacks block production model is described in [SIP-001][]. The standard outlines the mechanism by which elected
+block leaders can produce blocks on the Stacks blockchain either by batching transactions or by streaming them.
+Microblocks are the product of the streaming model.
+
+If a block leader has elected to mine microblocks, the leader selects transactions from the mempool and package them
+into microblocks during the current epoch. Transactions included in microblocks don't achieve finality until they're
+included in an anchor block, but can assumed to be highly likely to achieve finality in the order in which the leader
+packaged them.
+
+While there is a high degree of confidence that streamed blocks are finalized in the order in which a block leader
+selects them, there is still a chance that microblocks can be reorganized in an anchor block. The processing order of
+transactions in microblocks can change depending on on how the microblock is finalized in the anchor block.
+
+-> If a transaction is dependent on a chain state that could by altered by previous transactions with serious
+implications, you should carefully consider whether it should be performed using microblocks.
+
+## Enabling microblocks
+
+Miners can choose to enable or disable microblocks in their mining configuration. As a best practice, you should enable
+microblock mining. When an application or user submits a transaction, the transaction can include an argument that
+requires the transaction to be in a microblock, an anchor block, or in either.
+
+### Transactions
+
+Transactions include an option that controls if a miner should include them in microblocks or in anchor blocks. The
+[anchor mode][] transaction option is an optional argument that controls whether a transaction must be included in
+an anchor block or a microblock, or is eligible for either.
+
+### Mining
+
+Stacks miners must enable microblocks in their miner configuration to implement the block streaming model. For more
+information, see [mining microblocks][].
+
+## Developing with microblocks
+
+Stacks allows you to choose how any transaction should be included on the blockchain by the miners. This flexibility
+means you can submit transactions that require low latency processing for inclusion in microblocks, and require that
+high stakes transactions wait for anchor block finality.
```

Comment:
I think `highly order-dependent transactions` would be better than `high stakes`.

Reply:
🙏🏼